src/draw/shapes.typ

```diff
@@ -665,17 +666,21 @@
       (calc.max(from.at(0), to.at(0)), calc.max(from.at(1), to.at(1)))
     )
 
-    let style = styles.resolve(ctx.style, style)
-    if help-lines {
+    let style = styles.resolve(ctx.style, merge: style, root: "grid", base: (
```

Comment:
Any reason not having this in the default style dictionary?

Reply:
It doesn't really matter where the default of an element's style is. Really `styles.default` can just have stroke, fill, radius, padding, shorten and mark keys in it. Then use `base` for each element and it'll work just the same...